pgmpy/factors/CPD.py

```diff
@@ -218,11 +218,11 @@ def _repr_html_(self):
         evidence_card = self.cardinality[1:]
 
         if evidence:
-            evidence_card.reverse()
-            evidence_card.insert(0, 1)
+            evidence_card = evidence_card[::-1]     #reverse
+            evidence_card = np.insert(evidence_card, 0, 1)
             cum_card = np.cumprod(evidence_card)
             max_card = cum_card[-1]
-            evidence.reverse()
+            evidence = evidence[::-1]      #reverse
```

Comment:
@aelnouby Please remove the comment. No need to add comment since it's very clear from the code that you are reversing it.